### PR TITLE
feat: unit test added for page return component

### PR DESCRIPTION
**Pull request by nada-deriv**
## Changes:

Added unit test, and ts migration for page return component

### Screenshots:

Please provide some screenshots of the change.
Screenshot 2023-05-25 at 11 00 46 AM

### CODE REVIEW

File: packages/p2p/src/components/modal-manager/modals/filter-modal/filter-modal-header.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { observer } from 'mobx-react-lite';
 import { localize, Localize } from 'Components/i18next';
 import { useStores } from 'Stores';
-import PageReturn from 'Components/page-return/page-return.jsx';
+import PageReturn from 'Components/page-return';
```
Comment (nada-deriv) [inline packages/p2p/src/components/modal-manager/modals/filter-modal/filter-modal-header.jsx:5]: refactor of these sections not done. can be taken up when working on these files.

**Review by ameerul-deriv — APPROVED** [review]

File: packages/p2p/src/components/order-details/__tests__/order-details-wrapper.spec.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ describe('<OrderDetailsWrapper />', () => {
         render(<OrderDetailsWrapper {...desktop_props} />);
 
         await waitFor(() => {
-            expect(screen.getByTestId('order-details-wrapper-desktop')).toBeInTheDocument();
+            expect(screen.getByTestId('dt_page-return')).toBeInTheDocument();
```
Comment (niloofar-deriv) [inline packages/p2p/src/components/order-details/__tests__/order-details-wrapper.spec.js]:
```suggestion
            expect(screen.getByTestId('dt_page_return')).toBeInTheDocument();
```

The test ids name convention in our projects is this: `dt_lala_tada` :smile:

File: packages/p2p/src/components/page-return/page-return.tsx [hunk 3]
```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import classNames from 'classnames';
 import { Icon, Text } from '@deriv/components';
-import PropTypes from 'prop-types';
-import './page-return.scss';
 
-const PageReturn = ({ className, onClick, page_title }) => {
+type PageReturnProps = {
```
Comment (niloofar-deriv) [inline packages/p2p/src/components/page-return/page-return.tsx]:
```suggestion
type TPageReturn = {
```

Naming convention for our types: `TComponentName`

File: packages/p2p/src/components/page-return/page-return.tsx [hunk 4]
```diff
@@ -1,12 +1,16 @@
...
+    className?: string;
+    onClick: () => void;
+    page_title: string;
+};
+
+const PageReturn: React.FC<PageReturnProps> = ({ className = '', onClick, page_title }) => {
```
Comment (niloofar-deriv) [inline packages/p2p/src/components/page-return/page-return.tsx]: AFAIK we don't use `React.FC` in our project (I don't like this but it was recommended not using this)
```suggestion
const PageReturn = ({ className = '', onClick, page_title }: TPageReturn) => {
```

File: packages/p2p/src/components/page-return/page-return.tsx [hunk 5]
```diff
@@ -1,12 +1,16 @@
...
     return (
-        <div className={classNames('page-return', className)} data-testid='order-details-wrapper-desktop'>
+        <div className={classNames('page-return', className)} data-testid='dt_page-return'>
```
Comment (niloofar-deriv) [inline packages/p2p/src/components/page-return/page-return.tsx]:
```suggestion
        <div className={classNames('page-return', className)} data-testid='dt_page_return'>
```

**Review by farhan-nurzi-deriv — APPROVED** [review]

**Review by niloofar-deriv — APPROVED** [review]

**Pull request merged**